docs/client.rst

```diff
@@ -0,0 +1,137 @@
+The SecureDrop Client Application
+=================================
+
+As part of the ongoing work to make a user (journalist?) friendly workstation
+for SecureDrop we have created a native client application to be run within the
+Qubes operating system. It helps journalists with the most common activities
+associated with using SecureDrop in a user friendly manner.
+
+Currently the client is alpha quality although work is ongoing in terms of
+improving features and the user interface.
+
+The source code, and related issues are `hosted on GitHub <https://github.com/freedomofpress/securedrop-client>`_.
+
+Developer Setup
+---------------
+
+Clone the repository::
+
+    git clone git@github.com:freedomofpress/securedrop-client.git
+
+Create and enable a new Python3 virtualenv::
+
+    python3 -m venv my_virtualenv
+    source my_virtualenv/bin/activate
+
+Set up the Python dependencies::
+
+    pip install pipenv
+    pipenv install --dev
+    pipenv shell
+
+Install the packaged dependencies::
+
+    sudo apt install python3-pyqt5 python3-pyqt5.qtsvg xvfb
```

Comment:
I feel like running non-dev related commands in the `pipenv` should be discouraged because it's not necessary and also you lose things like history or some people might have non-standard setups this could conflict with.

I also feel like it's generally better to suggest doing `pipenv run make check` because if someone forgets to enter the dev shell, they are going to get confusing errors and then have to back track further to see where they went wrong.